addons/survey/static/src/js/survey_form.js

```diff
@@ -1169,7 +1173,16 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend(SurveyPreloa
         var self = this;
         var errorKeys = _.keys(errors);
         _.each(errorKeys, function (key) {
-            self.$("#" + key + '>.o_survey_question_error').append($('<p>', {text: errors[key]})).addClass("slide_in");
+            var msg = errors[key];
```

Comment:
Even if the rest of the file/method uses `var` you should in most cases instead use `let` when the value will be redefined (including when you split declaration and assignment if you consider an else statement for readability) and `const` otherwise. I'll let you find out which one applies here.
IMHO, you could also update the two other lines as you are rewriting most of the function.